.travis.yml

```diff
@@ -32,6 +32,10 @@ jobs:
       python: 3.7
       env: PURE_PYTHON=1
 
+    - name: "Python: 3.8, (forced C extensions)"
```

Comment:
comma parenthesis looks weird.

In previous jobs the parentheses were explaining the meaning of "pure".  I suggest no parentheses here: "Python: 3.8, forced C extensions"

Reply:
It occurs to me that on all the CPythons on CI we really want to force that the C extensions are built and working; they're not optional on CI. So I need to make that the default and only opt-out on PyPy.